test/distributed/_sharded_tensor/test_sharded_tensor.py

```diff
@@ -249,6 +259,105 @@ def test_full_with_dtype_overridden(self):
         expected_tensor = torch.full((h, w), fill_value=fill_value, device=local_device, dtype=torch.double)
         self.assertEqual(expected_tensor, local_tensor)
 
+
+class TestShardedTensorOps(ShardedTensorTestBase, MultiProcessTestCase):
+    def _run_sharded_linear(self, spec, input_size, linear_size, sharded_dim):
+        # Use same seed.
+        torch.manual_seed(0)
+        local_linear = torch.nn.Linear(*linear_size).cuda(self.rank)
+
+        sharded_linear = torch.nn.Linear(*linear_size)
```

Comment:
Could we also add a test case that  use this API to shard a module on a meta-device?